### PR TITLE
Github workflows

### PR DESCRIPTION
**Pull request by abzcoding**
We should have some sort of testing, to make sure nothing breaks LunarVim, this is just an attempt to do so,
I'm open to suggestions

@ChristianChiarulli  @rebuilt  @rafamadriz

### CODE REVIEW

**Comment by Tastyep** [comment]
Great idea !

File: .github/workflows/install.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    runs-on: ${{ matrix.runner }}
+    if: github.event.pull_request.draft == false
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: Install dependencies for Linux
+        if: matrix.os == 'linux'
+        run: |
+          sudo apt-get install -y gettext libtool libtool-bin autoconf automake cmake g++ pkg-config unzip build-essential
+          sudo bash ./utils/bin/install-latest-neovim
```
Comment (Tastyep) [inline .github/workflows/install.yaml:30]: We should not specify bash, if bash is required to run the script, then we should update the shebang

**Review by Tastyep — CHANGES_REQUESTED** [review]

**Comment by Tastyep** [comment]
While at it, could we add a formatting check ? `stylua --check` should do the job.

File: .github/workflows/install.yaml (hunk 1 above)
Comment (abzcoding) [reply .github/workflows/install.yaml:30]: Yeah, I think we should update it since we have `bash` in the readme for installation as well.

**Review by Tastyep — APPROVED** [review]

**Comment by abzcoding** [comment]
> While at it, could we add a formatting check ? `stylua --check` should do the job.

ok why do we have [this much](https://github.com/abzcoding/LunarVim/runs/3028225889?check_suite_focus=true) formatting errors :sob:

**Comment by abzcoding** [comment]
> > While at it, could we add a formatting check ? `stylua --check` should do the job.
> 
> ok why do we have [this much](https://github.com/abzcoding/LunarVim/runs/3028225889?check_suite_focus=true) formatting errors sob

#811 will address this :feelsgood:

File: .github/workflows/install.yaml (hunk 1 above)
Comment (abzcoding) [reply .github/workflows/install.yaml:30]: #803 has addressed this already

Comment (Tastyep) [reply .github/workflows/install.yaml:30]: Then we only need to do `sudo ./utils/bin/install-latest-neovim`, right ?

Comment (Tastyep) [reply .github/workflows/install.yaml:30]: While reviewing #803 I noticed a typo in the shebang of the installer, which is why you need to specify bash here.

**Comment by ChristianChiarulli** [comment]
@abzcoding do you think this is ready?

**Comment by abzcoding** [comment]
> @abzcoding do you think this is ready?

I think so, we should add new test cases later on

**Comment by luxus** [comment]
@ChristianChiarulli can this be merged?

**Pull request merged**